src/nyc_trees/nyc_trees/context_processors.py

```diff
@@ -10,23 +10,12 @@
 
 from django.utils.timezone import make_aware, utc
 
-from apps.survey.models import Blockface, Survey
+from apps.survey.models import (Blockface, Survey, Territory,
+                                BlockfaceReservation)
+from apps.users.models import TrustedMapper
 
 
 def settings(request):
```

Comment:
The original version of the ``settings`` context processor was named ``settings`` because all it did was poke setting through to the context. If we are going to expand it, I would rather give it a more accurate name, and undo the ``import settings as s`` aliasing.

Alternatively, ``settings`` could remain a stateless context processor and we move the request-based tiler logic into a separate function.